Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.10-slim-bullseye
```

Comment:
Maybe it's worth using here Python 3.9? Because the test system is still configured only for 3.9.

Reply:
done